app/src/routes/api/v1/geoStoreRouter.js

```diff
@@ -16,6 +17,8 @@ const router = new Router({
     prefix: '/geostore'
 });
 
+const LIMIT = 50;
```

Comment:
`LIMIT` of what? don't be afraid to use long names, do be afraid of us-in-6-months looking at this and not knowing what it does.

Also, this should probably go in the config files, not hardcoded here